content/docs/introduction/architecture-overview.md

```diff
@@ -24,3 +24,7 @@ Safekeepers can be thought of as an ultra-reliable write buffer that holds the l
 ## Durability
 
 Durability is at the core of Neon's architecture. As described earlier, incoming WAL data is initially stored across multiple availability zones in a [Paxos](<https://en.wikipedia.org/wiki/Paxos_(computer_science)>) cluster before being uploaded to a cloud object store, such as [Amazon S3](https://aws.amazon.com/s3/) (99.999999999% durability), both in raw WAL and materialized form. Additional copies are maintained across Pageservers to enhance the read performance of frequently accessed data. Consequently, there are always multiple copies of your data in Neon, ensuring durability.
+
+## Archive storage
+
+Archive storage in Neon, which enables [branch archiving](/docs/guides/branch-archiving) on the Free Plan, optimizes storage resources by offloading data that's not being used. As described above, Neon’s architecture includes Safekeepers, Pageservers, and cloud object storage. In this setup, the Pageservers are responsible for processing and uploading data to cloud object storage as soon as it's written. When a branch is archived, it does not involve moving data; instead, the branch's data is simply evicted from the Pageserver, freeing up the more costly Pageserver storage without affecting data durability or availability, as the data remains easily accessible in cloud object storage. This approach ensures that while archived data is readily available on demand, it's no longer taking up space in the higher-speed but less cost-efficient storage used by Neon's Pageservers.
```

Comment:
I believe we could be a little more terse:
```suggestion
Pageservers evict unused pages to ensure optimal utilisation of their premium storage. Because updates are always persisted to cloud object storage for durability, all data remains accessible on demand. [Branch archiving](/docs/guides/branch-archiving) optimises storage costs further by moving unused branches to cost-effective archival storage systems until they are requested.
```

Reply:
I've made this a little more succinct.